tutor/templates/addcourse.html

```diff
@@ -0,0 +1,31 @@
+{% extends "base.html" %}
+{% block title %} New Course {% endblock %}
+
+{% block content %}
+
+<div class="content=section">
+    <form method="POST" action="">
+        {{ form.hidden_tag() }}
+        <fieldset class="form-group">
+        <legend class="border-bottom mb-4">Add New Course</legend>
+            <div class="form-group">
+                {{ form.title.label(class="form-control-label") }}
+                {% if form.title.errors %}
```

Comment:
I would recommend indenting code that goes inside `if`s in templates just like you would in normal Python code so that its clear when the `if` block starts and where it ends.

Reply:
fixed